src/comment-pr/run.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+
+export async function run(): Promise<void> {
+  const githubToken = process.env.GITHUB_TOKEN;
+
+  if (!githubToken) {
```

Comment:
main functionality here

Reply:
functionality LGTM.

not familiar with this `github.context` but see that this is included in `@actions/github` lib (🙌 typescript)